builtin/providers/aws/resource_aws_api_gateway_method.go

```diff
@@ -107,6 +112,7 @@ func resourceAwsApiGatewayMethodCreate(d *schema.ResourceData, meta interface{})
 		RequestModels:     aws.StringMap(models),
 		RequestParameters: aws.BoolMap(parameters),
 		ApiKeyRequired:    aws.Bool(d.Get("api_key_required").(bool)),
+		AuthorizerId:      aws.String(d.Get("authorizer_id").(string)),
```

Comment:
True, we could do that, although based on tests it has the same effect as passing an empty string. Admittedly I'm in favour of changes that make resulting HTTP requests smaller so I'll update the PR. 😉 👍